test/performance/shmem_perf_suite/bi_dir.h

```diff
@@ -49,10 +53,55 @@ void static inline bi_bw(int len, perf_metrics_t *metric_info)
 
     } else {
         for (i = 0; i < metric_info->trials + metric_info->warmup; i++) {
-            for(j = 0; j < metric_info->window_size; j++)
+            for(j = 0; j < metric_info->window_size; j++) {
+#ifdef USE_NONBLOCKING_API
                 shmem_putmem(metric_info->dest, metric_info->src, len, dest);
+#else
+                shmem_putmem_nbi(metric_info->dest, metric_info->src, len, dest);
+#endif
+            }
+            shmem_quiet();
+        }
+    }
+}
+
+void static inline bi_bw_get(int len, perf_metrics_t *metric_info)
+{
+    double start = 0.0, end = 0.0;
+    int dest = partner_node(*metric_info);
+    int i = 0, j = 0;
+
+    shmem_barrier_all();
+
+    if (streaming_node(*metric_info)) {
+        for (i = 0; i < metric_info->trials + metric_info->warmup; i++) {
+            if(i == metric_info->warmup)
+                start = perf_shmemx_wtime();
 
+            for(j = 0; j < metric_info->window_size; j++) {
+#ifdef USE_NONBLOCKING_API
```

Comment:
need to switch getmem/getmem_nbi as well

Reply:
Thanks @kseager. These are corrected in my local version. Will check them in through my next commit.